src/screens/OtherProfile/index.js

```diff
@@ -157,7 +257,7 @@ const OtherProfile = () => {
     };
     const result = await setUnFollow(data);
     if (result.code === 200) {
-      refetchOtherProfile();
+      fetchOtherProfile();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchOtherProfile` function is called after unfollowing a user. Ensure that this function correctly updates the state and UI to reflect the change in follow status.
```diff
+ fetchOtherProfile();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->